### PR TITLE
feat: show button to verify vat ID

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

### CODE REVIEW

File: components/account/home/AccountInformation.vue [hunk 1]
```diff
@@ -93,10 +98,116 @@ async function update() {
...
+      else {
+        const timerVal = get(timer);
+        if (timerVal) {
+          clearInterval(timerVal);
+        }
+      }
+    }, 1000);
+    set(timer, newTimer);
+  }
+});
```
Comment (kelsos) [inline components/account/home/AccountInformation.vue:142]: I wonder if there is a better way to do the countdown

**Comment by kelsos** [comment]
@lukicenturi I did a quick check looks okish to me

What I am not 100% sure is the button to verify in the tooltip, but maybe we can discuss that tomorrow during the call with @AJAyanbisi

File: components/account/home/AccountInformation.vue (hunk 1 above)
Comment (lukicenturi) [reply components/account/home/AccountInformation.vue]: It's simplified now

Comment (kelsos) [reply components/account/home/AccountInformation.vue]: Thank you :)

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**